README.rst

```diff
@@ -152,6 +159,18 @@ The tests do not run with root or administrator privileges.
 
 Where ${USERNAME} is a user on the OS X machine, and ${PASSWORD} is the password in ``slaves.osx.passwords`` from the ``config.yml`` used to deploy the BuildBot master at ${MASTER}.
 
+For testing purposes, or if you do not have root privileges, run the following commands to start a build slave:
+
+.. code:: shell
+
+   # Set MASTER and PASSWORD as above
```

Comment:
This looks like it could be describing the commands below. Instead, put this in the prose outside of the comments.